AEMO-substrate/customSpec-dev.json

```diff
@@ -0,0 +1,98 @@
+{
```

Comment:
this is not required to commit this file to repository. Nodes are using raw format chainspec